Update/koto_002.txt

```diff
@@ -402,7 +402,7 @@ void main()
 
 //rリューン＝オークの律義な指摘にとぼけてみたが、…実際手入れが面倒臭くて怠っていたのが一番の理由なので、それ以上の言い訳を諦めた。
 	OutputLine(NULL, "リューン＝オークの律義な指摘にとぼけてみたが、…実際手入れが面倒臭くて怠っていたのが一番の理由なので、それ以上の言い訳を諦めた。",
-			NULL, "I tried playing dumb at Ryuun-Ohc's honest reminder... but I gave up on further excuses since the biggest reason was that I slacked off since actually tending to the blade was a pain in the ass.", Line_Normal);
+			NULL, "I tried playing dumb at Ryuun-Ohc's honest reminder... but I gave up on further excuses since the biggest reason was that I slacked off since actually tending the blade was a pain in the ass.", Line_Normal);
```

Comment:
```suggestion
			NULL, "I tried playing dumb at Ryuun-Ohc's honest reminder... but I gave up on further excuses since the biggest reason was that I slacked off, since actually tending to the blade was a pain in the ass.", Line_Normal);
```